tests/FunctionalJavascript/SettingsFormTest.php

```diff
@@ -263,6 +264,16 @@ function (NodeElement $element): string {
     $config = $this->loadConfig();
     $this->assertFalse($config->get('force_2fa'));
     $this->assertEquals([], $config->get('2fa_conditions'));
+
+    // Since we rely on a test module from core, and we also test that the
+    // following plugins don't show in the UI, we won't detect if the plugins
+    // have been removed. Add a safety check to look for their existence.
+    // This doesn't prevent core from changing completely their context
+    // definitions, but that's more unlikely.
```

Comment:
This comment is ok, but as a reader I would wish to get reminded of the context before the finer details.
```suggestion
    // The following conditions are provided by the 'condition_test' module, but are not shown in the form, because they don't match the criteria.
    // The assertions allow us to detect if the plugins get removed in a future version of Drupal core.
    // They will not tell is if their context definition changes, but that is less likely.
```